bindings/src/ketama.rs

```diff
@@ -19,6 +19,6 @@ impl KetamaPool {
     }
 
     fn get_slot(&self, key: &str) -> usize {
-        self.0.get_slot(key)
+        self.0.get_node_idx(key)
```

Comment:
Wouldn't we also need a way to get the server name back to python? We'll need the server name to satisfy the `get_node` interface for the hasher.